src/test/java/com/newestuser/imen/bgdate/BulgarianDateTest.java

```diff
@@ -25,4 +25,36 @@ private Date firstOfJanuary(int year) {
         calendar.set(Calendar.DAY_OF_MONTH, 1);
         return calendar.getTime();
     }
+
+    @Test
+    public void checkLeapYear() {
+        Date gregorianDate = secondOfJanuary(2020);
+        BulgarianDate bulgarianDate = new BulgarianDate(gregorianDate);
+        String formatedDate = bulgarianDate.format(bulgarianDate);
+        assertEquals(formatedDate, "11 Алем, 6788 Сомор");
+    }
+
+    private Date secondOfJanuary(int year) {
```

Comment:
We already start to get this code duplicated. You can refactor the existing private method to accept 3 parameters `day`, `month` and `year` and reuse it in all the tests.

Example
```java
  @Test
    public void checkNonCenturialLeapYear() {
        Date gregorianDate = newDate(3, Calendar.MARCH, 1800);
        BulgarianDate bulgarianDate = new BulgarianDate(gregorianDate);
        String formatedDate = bulgarianDate.format(bulgarianDate);
        assertEquals(formatedDate, "10 Читем, 6568 Бечин");
}
```

Also please move the private methods below the tests. They are more of a helper methods when reading the tests one should not care that much of those helper methods.